src/views/UsersView.tsx

```diff
@@ -95,66 +101,95 @@ export const UsersView = (props: Props): JSX.Element => {
       </List>
     );
   } else {
-    pendingInvites = <>No pending invites</>;
+    pendingInvites = (
+      <Typography style={{ marginTop: "10px", marginBottom: "15px" }}>
+        No pending invites
+      </Typography>
+    );
   }
 
   const inviteForm = (
-    <form autoComplete="off" onSubmit={() => inviteNewUser}>
-      <div>{inviteMessage}</div>
-      <TextField
-        id="invite-email"
-        type="email"
-        required
-        onChange={handleChange}
-        value={inviteEmail}
-      />
-      <IconButton type="submit" onSubmit={(e) => e.preventDefault()}>
-        <Send />
-      </IconButton>
-    </form>
+    <>
+      <Typography>Why not invite someone else to collaborate?</Typography>
+      <form autoComplete="off" onSubmit={() => inviteNewUser}>
+        <div>{inviteMessage}</div>
+        <TextField
+          id="invite-email"
+          type="email"
+          required
+          onChange={handleChange}
+          value={inviteEmail}
+        />
+        <IconButton type="submit" onSubmit={(e) => e.preventDefault()}>
+          <Send />
+        </IconButton>
+      </form>
+    </>
   );
 
   return (
-    <>
-      <Grid container direction="row">
-        <Card>
-          <Paper
-            elevation={0}
-            variant="outlined"
-            square
-            className={classes.paperHeader}
-          >
-            <Typography className={classes.topography}>
-              Current Users
-            </Typography>
-          </Paper>
-          <Paper elevation={0} square className={classes.paperBody}>
-            <List>
+    <div style={{ display: "flex" }}>
+      <div
+        style={{
+          flexGrow: 0,
+          flexShrink: 0,
+          marginLeft: "20px",
+          marginRight: "20px",
+        }}
+      >
+        <PageSelector page="users" />
+      </div>
+      <Card
+        className={classes.usersCard}
+        style={{ width: "70%", height: "90vh" }}
+      >
+        <Paper
+          elevation={0}
+          variant="outlined"
+          square
+          className={classes.paperHeader}
+        >
+          <Typography className={classes.topography}>Current Users</Typography>
+        </Paper>
+        <TableContainer>
+          <Table aria-label="simple table">
+            <TableHead>
+              <TableRow>
+                <TableCell>Name</TableCell>
+                <TableCell>Email</TableCell>
+              </TableRow>
+            </TableHead>
+            <TableBody>
               {team?.profiles.map(({ email, name }) => (
-                <ListItem key={email}>{name}</ListItem>
+                <TableRow key={email}>
```

Comment:
Do we have a way to remove a user from a team? If not, can we make an issue?

Reply:
Will make an issue.